### PR TITLE
new drink

### PR DESCRIPTION
**Pull request by PrajapatiAnkit**
new drink added, please merge this PR

### CODE REVIEW

File: src/recipes/concrete-junglebird.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            "quantity": "0.75",
+            "measure": "oz",
+            "ingredient": "chai infused Jameson Black Barrel"
+        }
+    ],
+    "directions": [
+        "Add all ingredients to a mixer with ice",
+        "Shake vigorously, strain into a double rocks glass and garnish with a pineapple leaf."
+    ],
+    "image": https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/img-2868-1546986659.jpg?crop=0.444581154516559xw:1xh;center,top&resize=980:*",
```
Comment (alfg) [inline src/recipes/concrete-junglebird.json]: Please add the image and use `concrete-junglebird.jpg` here.

**Comment by alfg** [comment]
Drink should be 600x400px, with 600px being the width.

File: src/recipes/concrete-junglebird.json (hunk 1 above)
Comment (PrajapatiAnkit) [reply src/recipes/concrete-junglebird.json]: hi there, image added. Now can I get this PR merged?
thanks !

Comment (PrajapatiAnkit) [reply src/recipes/concrete-junglebird.json]: drink image updated with 600x400px

Comment (alfg) [reply src/recipes/concrete-junglebird.json]: Was this image stretched?

Comment (alfg) [reply src/recipes/concrete-junglebird.json]: It needs to be 600x400px maintaining the aspect ratio. If you need to, you can add a white border canvas around the image to maintain the aspect ratio.

**Comment by alfg** [comment]
600px should be the width, please.

**Comment by alfg** [comment]
Closing. Feel free to reopen with updated image.

**Pull request closed without merging**